assets/js/main.js

```diff
@@ -83,6 +83,21 @@
 
 		}
 
+	// Cookie Banner.
+	$(document).ready(function(){
+		if (document.cookie.indexOf('cookie-notice=true') < 0) {
+			setTimeout(function () {
+				$("#cookieConsent").fadeIn(200);
+			 }, 2000);
+
+			$("#closeCookieConsent, .cookieConsentOK").click(function() {
+				document.cookie = 'cookie-notice=true; expires=Thu, 31 Dec 2099 23:59:59 UTC; path=/'

```

Comment:
Hier wird der Cookie platzier, wenn es noch keinen gibt, richtig? Und er bleibt auf ewig (bis 2099) auf dem Gerät, sodass er beim zweiten Besuch der Seite icht mehr angezeigt wird, oder? Ist das in ordnung so, oder müssen wir bei jedem Besuch informieren?